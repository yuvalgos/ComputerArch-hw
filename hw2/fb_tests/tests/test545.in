w 0x7bc24
w 0x7bc74
w 0x7bc54
w 0x7bc90
r 0x7bc08
w 0x7bc30
w 0x7bc78
r 0x7bc20
w 0x7bc68
r 0x7bc24
w 0x7bc88
r 0x7bc80
w 0x7bc7c
r 0x7bc48
w 0x7bc40
r 0x7bc00
r 0x7bbfc
w 0x7bc60
w 0x7bc30
w 0x7bc88
w 0x7bbf4
w 0x7bc84
r 0x7bc44
w 0x7bc44
w 0x7bbf4
r 0x7bc5c
r 0x7bc00
r 0x7bc58
w 0x7bc90
w 0x7bc48
r 0x7bc40
w 0x7bc78
r 0x7bc04
r 0x7bc14
w 0x7bc70
r 0x7bc40
w 0x7bc08
w 0x7bc54
w 0x7bc5c
w 0x7bc20
w 0x7bc0c
