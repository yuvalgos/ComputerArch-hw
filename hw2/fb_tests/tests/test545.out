L1miss=0.878 L2miss=0.944 AccTimeAvg=174.073
